sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -138,13 +132,7 @@ export function record(
     testContext.skip();
   }
 
-  setEnvironmentOnLoad();
-
-  if (isBrowser()) {
-    recorder = new NiseRecorder(currentHash, testHierarchy, testTitle);
-  } else {
-    recorder = new NockRecorder(currentHash, testHierarchy, testTitle);
-  }
+  const recorder = createRecorder(currentHash, testHierarchy, testTitle);
```

Comment:
The changes look great!
Can you please do some sanity testing so that we don't have regressions?

- Pick storage-blob(and TA)
- Set .only for a couple of tests 
- Set the mode as "record" and generate recordings
- Run the same tests again in "playback" to make sure it works
- Do this in both node and browser

Reply:
@HarshaNalluru I ran Form Recognizer tests and the BlobClient tests, and found no errors.

Reply:
Perfect!!